### ISSUE TITLE
Can't configure xcode resource to only install GM version

### ISSUE THREAD

**Issue by netzere** [microsoft/macos-cookbook#221]
## Describe the Bug 
Not sure if i'd consider it as a bug, 
but i want to use this cookbook to only install GM versions of xcode. 
When specifying '11.5' to the xcode resource, it installs the beta version.
I wasn't able to find a way to configure the resource so it downloads only GM.

## Error Message  
None

## Chef Exception  
None

**Comment by jazaval** [comment]
Hey @netzere - this looks to be caused an issue with the upstream `xcode-install` gem used to retrieve Xcode from Apple. There's an [open PR](https://github.com/xcpretty/xcode-install/pull/393) to address the issue. In the meantime, if you need to work around this, I recommend caching a copy of the 11.5 release and use the `download_url` property to avoid the resource grabbing the wrong version from Apple.

**Comment by netzere** [comment]
Hi @jazaval - the PR is now merged.
Do you need to release a new version as well? or should it work now?

**Comment by netzere** [comment]
Hi @jazaval - i see that the xcode-install PR is now merged.
I tried to run my pipeline and i see the cookbook downloaded the new version (2.6.5),
however it still downloads the beta version instead of GM.

```
 Successfully installed aws-sdk-kms-1.34.1
 ​Successfully installed aws-sdk-s3-1.68.1
 ​Successfully installed fastlane-2.149.1
 ​Successfully installed xcode-install-2.6.5
 ​58 gems installed
[2020-06-15T01:57:34-07:00] INFO: execute[install xcode gem] ran successfully
     ​- execute curl -sL -O https://github.com/neonichu/ruby-domain_name/releases/download/v0.5.99999999/domain_name-0.5.99999999.gem && /opt/chef/embedded/bin/gem install --no-document domain_name-0.5.99999999.gem && /opt/chef/embedded/bin/gem install --no-document --conservative xcode-install && rm -f domain_name-0.5.99999999.gem
   ​
 ​
 ​* xcode[11.5] action install_xcode[2020-06-15T01:57:34-07:00] INFO: Processing xcode[11.5] action install_xcode (chef-mac::xcode line 3)

   ​* execute[install Xcode 11.5 beta] action run[2020-06-15T01:57:46-07:00] INFO: Processing execute[install Xcode 11.5 beta] action run (/Users/***/*****/chef/conf/local-mode-cache/cache/cookbooks/macos/resources/xcode.rb line 41)
```




Any help?

**Comment by jazaval** [comment]
Hey @netzere, is this running on a node that already installed the 11.5 beta with the previous `xcode-install` gem? Also, can you provide your macOS and chef-client versions? I haven't been able to repro this yet.

**Comment by jazaval** [comment]
Nevermind - I've opened a branch that has some new failing tests that should isolate this. Working on it now.

**Comment by netzere** [comment]
@jazaval Great! any update regarding this issue?

**Comment by jazaval** [comment]
I did resolve the issue and got the correct version installing on this [branch](https://github.com/microsoft/macos-cookbook/tree/hotfix/xcode), however there's a [new issue](https://github.com/xcpretty/xcode-install/issues/396) with the `xcode-install` gem in `2.6.5`. I've written a quick fix for that issue and have opened a [PR](https://github.com/xcpretty/xcode-install/pull/397) on that repo. Waiting on a maintainer response now.
